tests/test_ui_extra_new.py

```
@@ -0,0 +1,185 @@
+            m.props.return_value.tooltip.return_value = m
+            return m
+        mock_ui.button.side_effect = mock_button
+
+        ui_inst.draw_editor("root")
+
+        assert 'switch' in captured_cbs
+        e = MagicMock()
+        e.value = True
+        captured_cbs['switch'](e)
+
+        assert getattr(ui_inst, '_is_hex_int_val_root') is True
```

Ruff (0.15.13)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)